Extension/browser/firefox/lib/prefs.js

```diff
@@ -36,6 +36,10 @@ var Prefs = exports.Prefs = {
 		var url = "filters/filter_" + filterId + ".txt";
 		return self.data.url(url);
 	},
+	getLocalMobileFilterPath: function (filterId) {
+		var url = "filters_mobile/filter_" + filterId + ".txt";
```

Comment:
At least leave them in the same folder, just change naming to "filter_mobile_" + filterId